### DISCUSSION TITLE
problems marshaling `null.Bytes` field containing uuid via json.Marshal

### DISCUSSION THREAD

**Discussion by superpan** [aarondl/sqlboiler#1111] [Q&A] [answered]
I have a MySQL table with a nullable binary field for optional foreign keys to an UUID field.  When that field is populated with a valid binary value, calling json.Marshal fails due to invalid character error from marshaling `null.Bytes`.  Have anyone ran into the same issue?

**Comment by stephenafamo** [comment, marked as answer]
Since a UUID is not valid JSON, I don't think using json.Marshal is the way to go.
